h2/src/main/org/h2/value/ValueEnum.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2004-2014 H2 Group. Multiple-Licensed under the MPL 2.0,
+ * and the EPL 1.0 (http://h2database.com/html/license.html).
+ * Initial Developer: H2 Group
+ */
+package org.h2.value;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+import org.h2.engine.SysProperties;
+import org.h2.util.MathUtils;
+import org.h2.util.StringUtils;
+
+/**
+ * Implementation of the ENUM data type.
+ */
+public class ValueEnum extends ValueString {
```

Comment:
because you are still extending ValueString, I'm pretty sure the storage on disk is going to store string values instead of integers